eventq_rabbitmq/lib/eventq_rabbitmq/rabbitmq_queue_worker.rb

```diff
@@ -28,7 +28,7 @@ def start(queue, options = {}, &block)
         raise "[#{self.class}] - Worker is already running." if running?
 
         if options[:client] == nil
-          raise "[#{self.class}] - :client (QueueClient) must be specified."
+          EventQ.log(:info, "[#{self.class}] - options[:client] is now deprecated!!, please pass options[:aws_account_no] && options[:aws_region].")
```

Comment:
the comment here in the log message is about `aws_region` which is for aws and not rabbitmq.  But this is the rabbitmq worker.